asctb-api/src/functions/api.functions.ts

```diff
@@ -90,6 +90,21 @@ function addREF(rowHeader: any, newRow: any, key: any, data: any) {
   }
 }
 
+export function fetchMetaData(data:any[]): Promise<{}>{
```

Comment:
This does not need to be wrapped as a promise (not sure why it was in the first place). Also it's not implemented well either. This metadata should be returned from the makeASCTBData function below (plus the warnings for #353). Try to follow more closely the model in the updated makeASCTBData function.